From d4a8819d98503ec508f74e457f5136be3bdb32fc Mon Sep 17 00:00:00 2001
From: Rex Dieter <rdieter@gmail.com>
Date: Thu, 9 Aug 2018 16:12:33 +0200
Subject: [PATCH] configure.py: make check for PyQt5.sip module non-fatal

Python fails to load this if PyQt5 is not available, so... bootstrapping
fails.

Since this is only really required at runtime anyway, make the check
itself informative only.
---
 configure.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.py b/configure.py
index 7c48136..5e05433 100644
--- a/configure.py
+++ b/configure.py
@@ -646,7 +646,7 @@ class TargetConfiguration:
         try:
             from PyQt5 import sip
         except ImportError:
-            error(
+            inform(
                     "Unable to import PyQt5.sip.  Make sure you have "
                     "configured SIP to create a private copy of the sip "
                     "module.")
@@ -2918,7 +2918,7 @@ def check_sip(target_config):
     try:
         from PyQt5 import sip
     except ImportError:
-        error(
+        inform(
                 "Unable to import PyQt5.sip.  Make sure you have configured "
                 "SIP to create a private copy of the sip module.")
 
-- 
2.17.1

